src/studio/src/designer/backend/Controllers/PreviewController.cs

```diff
@@ -0,0 +1,247 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Interface.Models;
+using Altinn.Studio.Designer.Helpers;
+using Altinn.Studio.Designer.Infrastructure.GitRepository;
+using Altinn.Studio.Designer.RepositoryClient.Model;
+using Altinn.Studio.Designer.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Studio.Designer.Controllers
+{
+    /// <summary>
+    /// Controller containing all actions related to data modelling
+    /// </summary>
+    [Authorize]
+    [AutoValidateAntiforgeryToken]
+    [Route("preview/{org}/{app}")]
+    public class PreviewController : Controller
+    {
+        private readonly IRepository _repository;
+        private readonly ISchemaModelService _schemaModelService;
+        private readonly IAltinnGitRepositoryFactory _altinnGitRepositoryFactory;
+
+         /// <summary>
+        /// Initializes a new instance of the <see cref="PreviewController"/> class.
+        /// </summary>
+        /// <param name="repositoryService">The service repository service.</param>
+        /// <param name="sourceControl">The source control service.</param>
+        /// <param name="altinnGitRepositoryFactory">
+        /// Factory class that knows how to create types of <see cref="AltinnGitRepository"/>
+        /// </param>
+        public PreviewController(
+            IRepository repositoryService, ISourceControl sourceControl, IAltinnGitRepositoryFactory altinnGitRepositoryFactory)
+        {
+            _repository = repositoryService;
+            _altinnGitRepositoryFactory = altinnGitRepositoryFactory;
+        }
+
+        /// <summary>
+        /// The index action which will show the React form builder
+        /// </summary>
+        /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
+        /// <param name="app">Application identifier which is unique within an organisation.</param>
+        /// <returns>A view with the React form builder</returns>
+        public IActionResult Index(string org, string app)
```

Comment:
This method can be deleted, as it loads the .cshtml view that is no longer relevant.